### DISCUSSION TITLE
Documentation for private functions

### DISCUSSION THREAD

**Discussion by chechu-arias** [mkdocs/mkdocs#3148] [Q&A] [answered]
Hi, I would like to add to the documentation of Mkdocs my private functions without changing the function name. Can I do this? And if so, how would I do that?

**Comment by pawamoy** [comment, marked as answer]
You might have found out this is actually related to mkdocstrings and not MkDocs, and the way to do that is to change/reset the default filters, documented here: https://mkdocstrings.github.io/python/usage/#globallocal-options (search for "filters").

**Reply by chechu-arias** [reply]
Thank you for your time!
I found an answer looking at mkdocstrings as you pointed out and now it's working.
